keras_preprocessing/image/image_data_generator.py

```diff
@@ -596,6 +597,7 @@ def flow_from_dataframe(self,
                 `model.predict_generator()`, `model.evaluate_generator()`, etc.).
             batch_size: size of the batches of data (default: 32).
             shuffle: whether to shuffle the data (default: True)
+                If set to False, sorts the data in alphanumeric order.
```

Comment:
Please remove as this returns a `DataFrameIterator`

Reply:
Done